src/modules/netinstall/PackageModel.cpp

```diff
@@ -170,6 +170,38 @@ PackageModel::headerData( int section, Qt::Orientation orientation, int role ) c
     return QVariant();
 }
 
+/** @brief Sets the checked flag on matching groups in the tree
```

Comment:
Documentation would typically go in the header. I know for modules which don't install the headers at all this is less of an issue, but I'd typically look in the header **first** for explanatory comments.